data/templates/Detector_AzureStackHub.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
```

Comment:
Since the method is passing the KustoClusterName and KustoDataBaseName, there is no need to write the query as such. Will modify the onboarding document template.